lib/screen/global_receive/receive_detail_page.dart

```diff
@@ -69,68 +70,81 @@ class _GlobalReceiveDetailPageState extends State<GlobalReceiveDetailPage> {
                     )),
               ),
               const SizedBox(height: 48),
-              Text((_blockchainNFTText(_account.blockchain)),
-                  style: theme.textTheme.headlineMedium),
-              accountItem(context, _account),
-              GestureDetector(
-                  onTap: copy,
-                  child: Container(
-                    padding: const EdgeInsets.fromLTRB(8, 8, 8, 16),
-                    decoration: const BoxDecoration(
-                      border: Border(
-                        top:
-                            BorderSide(color: Color.fromRGBO(227, 227, 227, 1)),
+              Container(
```

Comment:
Can you replace this block with `ImportantNoteView(note: _blockchainWarningText(_account.blockchain))`